keps/sig-scheduling/5147-dra-node-reservations/kep.yaml

```diff
@@ -0,0 +1,24 @@
+title: Production Readiness Review Process
+kep-number: 5147
+authors:
+  - "@dom4ha"
+owning-sig: sig-scheduling
+status: implementable
+creation-date: 2025-02-05
+participating-sigs:
+  - sig-release
+reviewers:
+  - 
+approvers:
+  - 
+latest-milestone: '1.33'
+stage: alpha
+
+milestone:
+  alpha: "v1.33"
+
+feature-gates:
+  - name: DraNodeReservations
+    components:
+      - kube-scheduler
```

Comment:
I'm still not fully convinced this is a scheduler area... But @johnbelamaric can prove me wrong :sweat_smile: